### PR TITLE
Including *.json files to fix #725

### LINKED ISSUE #725: JRPG demo softlocks when interacting with the yellow "o" item (null textbox content)
**Which demo project is affected:**
JRPG Demo on the web browser
https://godotengine.github.io/godot-demo-projects/2d/role_playing_game/

**OS/device including version:**
Windows 10 x64
Vivaldi 5.2.2623.39 (Stable channel) (64 bits) 

**Issue description:**
When interacting with the yellow "o" box, the following dialogue box displays "Null Null" and no input can make the player escape the dialogue interaction. The animation for the space or enter key being pressed is still played on the big yellow "Next" button, but it still doesn't closes the dialogue or allows you to move and test the rest of the demo

**Screenshots of issue:**

### PR DESCRIPTION
**Pull request by voylin**
There is a problem with the automatically exported HTML5 demos. When they use json files for dialogue by example, as in the JRPG demo, the game gets errors as it can't load those specific *.json files.

### CODE REVIEW

**Comment by aaronfranke** [comment]
I haven't tested this, but it makes sense.

**Comment by voylin** [comment]
> I haven't tested this, but it makes sense.

Sorry, I forgot to put this into a branch with a more fitting name. ^^"

File: .github/dist/export_presets.cfg [hunk 1]
```diff
@@ -1,11 +1,11 @@
-[preset.0]
+﻿[preset.0]
```
Comment (aaronfranke) [inline .github/dist/export_presets.cfg:1]: You added a BOM here, it should be removed.

Comment (voylin) [reply .github/dist/export_presets.cfg:1]: I don't know what went wrong here, GitKraken didn't show this when I pushed the commit. I'll fix it ASAP!

**Review by aaronfranke — APPROVED** [review]

**Comment by voylin** [comment]
Fixed it, still not sure what happened to that first line ^^"

**Comment by aaronfranke** [comment]
It looks like it's deployed, but the JRPG demo still has "Null" in the dialogue.

**Comment by voylin** [comment]
> It looks like it's deployed, but the JRPG demo still has "Null" in the dialogue.

@aaronfranke Try refreshing your browser or try in incognito. For me the demo is working perfectly right now when using the link. Does that fix it?

**Comment by aaronfranke** [comment]
Yup, it just took awhile to show up. It's working now. Never mind, false alarm.

**Pull request merged**